trussworks/react-uswds
# build(dev-deps): upgrade linting deps

# Summary

Upgrade eslint and other build time dependencies. Note this PR depends on #2200 and should be merged after that one.

## How To Test

Ensure that `yarn install`, `yarn lint`, and `yarn test` pass successfully.

Ensure that the Breadcrumb stories still render the same in storybook.
